docs/modules/ROOT/pages/how-to-guides/proc_manually_triggering_a_release_plan.adoc

```diff
@@ -0,0 +1,56 @@
+= Manually Triggering a ReleasePlan
+
+This document covers the process to manually trigger a ReleasePlan for a given Snapshot of an Application when using Konflux. Common use cases include re-releasing a snapshot that failed to release due to a now-resolved condition or in the case that a ReleasePlan is not automatically triggered on every Snapshot intentionally to enforce manual releases (see: Releases to Production Registries that happen infrequently).  
```

Comment:
This is all great info, and it's super helpful to have use cases. But I recommend breaking it up into shorter sentences to not overwhelm readers, and rephrasing a bit for clarity:
"One common use case for this procedure is re-releasing a snapshot that failed to release due to an issue that has been resolved. Another use case is when manual releases are mandatory, because ReleasePlans are  not automatically triggered on every snapshot. (See: Releases to production registries that happen infrequently)."

Also, should that sentence in the parentheses have a link to something? ^^